docs/examples/en/controls/TrackballControls.html

```diff
@@ -171,6 +171,11 @@ <h3>[property:Boolean staticMoving]</h3>
 			Whether or not damping is disabled. Default is `false`.
 		</p>
 
+		<h3>[property:Vector3 target]</h3>
+		<p>
+			The point in world space the controls are centered around.
```

Comment:
How about using a similar wording like in `OrbitControls` so it can be just:
> The focus point of the controls.

Reply:
Sounds good to me!